pytest_rts/utils/common.py

```diff
@@ -43,9 +46,28 @@ def get_existing_tests(coverage_file_path: str) -> Set[str]:
 
 
 def get_tests_from_changes(
-    file_diffs: Dict[str, str], coverage_file_path: str
+    commithash_to_compare: str, coverage_file_path: str
 ) -> Set[str]:
-    """Returns the test set from given Git diffs"""
+    """Returns the test set from Git changes.
+    If the provided commithash does not exist in the repo,
+    then only changes in the git working directory are considered.
+    Otherwise the commithash is compared to the current working copy.
+    """
+    repo = get_git_repo()
+    if not commit_exists(commithash_to_compare, repo):
```

Comment:
Inverted condition is more readable: compare `if commit_exists()` with `if not commit_exists()`